roles/vendors/supermicro/tasks/eject.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I don't think these should be extracted into the same file. There should at least be a flag to select which tasks of which version of the API gets run.

Reply:
@nocturnalastro I added a conditional to execute one or another if the API is available. Please take a look and let me know.